CHANGELOG.md

```diff
@@ -3,12 +3,11 @@
 ## Bug fixes
 
 * Position textarea label at the top of the component
+* Prevent hover styles being applied when hovering over a readonly or a disabled `input` element.
 
 # 3.2.3
 
-## Bug Fixes
-
-* Prevent hover styles being applied when hovering over a readonly or a disabled `input` element.
+Merged in v3.1.6
```

Comment:
minor - is this needed?

Reply:
Yep: https://github.com/Sage/carbon/blob/release/CHANGELOG.md#323